### PR TITLE
JAMES-2770 RRT rewriting can lead to duplicated mails

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/AliasMappingTest.java [hunk 1]
```diff
@@ -370,4 +370,23 @@ public void messageShouldBeStoredInRepositoryWhenAliasLoopMapping() throws Excep
                 .getRepositoryMailCount(RRT_ERROR_REPOSITORY) == 1);
     }
 
+    @Test
+    public void userSHouldNotReceiveDuplicatesWhenUserAndAliasRegisteredToAGroup() throws Exception {
```
Comment (aduprat) [inline server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/AliasMappingTest.java]: s/SHould/Should

**Comment by chibenwa** [comment]
test this please

**Comment by Arsnael** [comment]
```
[819c7af290361d6f642a767b0510f3ea3b60c98c] [ERROR] Error occurred in starting fork, check output in log
[819c7af290361d6f642a767b0510f3ea3b60c98c] [ERROR] Process Exit Code: 100
[819c7af290361d6f642a767b0510f3ea3b60c98c] [ERROR] Crashed tests:
[819c7af290361d6f642a767b0510f3ea3b60c98c] [ERROR] org.apache.james.webadmin.routes.UserQuotaRoutesTest$ElasticSearchGetUsersQuotaRouteTest
[819c7af290361d6f642a767b0510f3ea3b60c98c] [ERROR] org.apache.maven.surefire.booter.SurefireBooterForkException: ExecutionException The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
```
test this please

**Comment by chibenwa** [comment]
```
[819c7af290361d6f642a767b0510f3ea3b60c98c] [ERROR] Failures: 
[819c7af290361d6f642a767b0510f3ea3b60c98c] [ERROR]   RabbitMQEventBusTest$LifeCycleTest$SingleEventBus.dispatchShouldWorkAfterNetworkIssuesForOldRegistration:323 
[819c7af290361d6f642a767b0510f3ea3b60c98c] Wanted but not invoked:
[819c7af290361d6f642a767b0510f3ea3b60c98c] mailboxListener.event(
[819c7af290361d6f642a767b0510f3ea3b60c98c]     org.apache.james.mailbox.events.MailboxListener$MailboxAdded@bf1a1130
[819c7af290361d6f642a767b0510f3ea3b60c98c] );
[819c7af290361d6f642a767b0510f3ea3b60c98c] -> at org.apache.james.mailbox.events.RabbitMQEventBusTest$LifeCycleTest$SingleEventBus.dispatchShouldWorkAfterNetworkIssuesForOldRegistration(RabbitMQEventBusTest.java:323)
[819c7af290361d6f642a767b0510f3ea3b60c98c] Actually, there were zero interactions with this mock.
```

test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**